release-drafter.yml

```diff
@@ -0,0 +1,17 @@
+name-template: v$NEXT_MINOR_VERSION 🌈
+tag-template: configuration-as-code-$NEXT_MINOR_VERSION
+categories:
+  - title: 🚀 Features
+    label: feature
+  - title: 🐛 Bug Fixes
+    label: fix
+  - title: 👻 Maintenance
+    label: chore
+  - title: 📝 Documentation updates
+    label: documentation
+  - title: 🚦 Tests
+    label: test
+
+template: |
+  ## Changes
+  $CHANGES
```

Comment:
I am pretty sure you want a space between headers :sweat:

Reply:
that what you meant?

Reply:
yup :sweat_smile:

Reply:
I updated my first comment :neutral_face: